pages/demo-get-involved.vue

```diff
@@ -0,0 +1,237 @@
+<template>
```

Comment:
Why `demo-get-involved`?

Reply:
Because it is a demo/development version of the already made get-involved page

Reply:
in your demo you should replace the existing page